components/apimgt/org.wso2.carbon.apimgt.rest.api.authenticator/src/main/java/org/wso2/carbon/apimgt/rest/api/authenticator/internal/ConfigurationActivator.java

```diff
@@ -0,0 +1,44 @@
+package org.wso2.carbon.apimgt.rest.api.authenticator.internal;
+
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.component.annotations.ReferenceCardinality;
+import org.osgi.service.component.annotations.ReferencePolicy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.wso2.carbon.kernel.configprovider.ConfigProvider;
+
+/**
+ * Class used to activate configuration loading
+ */
+
+@Component(
+        name = "org.wso2.carbon.apimgt.authenticator",
+        immediate = true
+)
+
+public class ConfigurationActivator {
+    private static final Logger log = LoggerFactory.getLogger(ServiceReferenceHolder.class);
```

Comment:
Incorrect Log object initialization .It has passed ServiceReferenceHolder.class instead of ConfigurationActivator.class . It should be as follows. 

private static final Logger log = LoggerFactory.getLogger(ConfigurationActivator.class);